src/app/api/_api-services/redis_service/index.ts

```
@@ -597,4 +602,31 @@ export class RedisService {
+	}: {
+		network: string;
+		proposalType: string;
+		indexOrHash: string;
+		data: IPostAnalytics;
+	}): Promise<void> {
+		const isActivePost = data.proposal.status && ACTIVE_PROPOSAL_STATUSES.includes(data.proposal.status as EProposalStatus);
+
+		await this.Set({
+			key: this.redisKeysMap[ERedisKeys.POST_ANALYTICS_DATA](network, proposalType, indexOrHash),
+			value: JSON.stringify(data),
+			ttlSeconds: isActivePost ? HALF_HOUR_IN_SECONDS : THREE_DAYS_IN_SECONDS
```

ESLint

[error] 'THREE_DAYS_IN_SECONDS' is not defined.

(no-undef)